field.go

```diff
@@ -151,6 +162,8 @@ func (f Field) addTo(kv KeyValue) error {
 		err := f.obj.MarshalLog(kv)
 		closer.CloseField()
 		return err
+	case unsafeBytesType:
+		kv.UnsafeAddBytes(f.key, []byte(f.str))
```

Comment:
We're taking in `[]byte`, copying to make a `string` to store in the Field, then copying again to make a `[]byte` that is appended. Ideally we:
 - only use a single type all the way. since we want to use fields in the string, maybe it should just be string?
 - or we have a second `UnsafeString` version to avoid all the copying.